freqtrade/exchange/exchange.py

```diff
@@ -1822,6 +1889,46 @@ def set_margin_mode(self, pair: str, collateral: Collateral, params: dict = {}):
         except ccxt.BaseError as e:
             raise OperationalException(e) from e
 
+    @retrier
+    def _get_mark_price_history(self, pair: str, since: int) -> Dict:
```

Comment:
I think something must've gone wrong with the rebase you (probably) did - probably while resolving some merge conflicts.

I've removed this code (as well as `get_funding_rate_history()`) in #6043 (as well as the corresponding test) - as there's a much simpler (and more efficient) way to accomplish this.